examples/docker.yaml

```diff
@@ -61,4 +61,4 @@ probes:
     hint: See "/var/log/cloud-init-output.log". in the guest
 portForwards:
   - guestSocket: "/run/user/{{.UID}}/docker.sock"
-    hostSocket: "{{.Home}}/docker.sock"
+    hostSocket: docker
```

Comment:
"docker" may look like either a reserved word or an abstract file path in the "current" directory.

Can we rather use a form like `hostSocket: "${{.InstanceDir}}/sock/docker"` ?

Reply:
I think "docker.sock" is quite established by now